cpp/include/cudf/structs/structs_column_view.hpp

```diff
@@ -30,7 +30,7 @@ namespace cudf {
  * @{
  */
 
-class structs_column_view : private column_view {
+class structs_column_view : public column_view {
```

Comment:
It's needed so that someone can do this:

structs_column_view scv(col_view);
scv.num_children()     <--- can't access this unless column_view is exposed as public.

Reply:
Or you just add a `using column_view::num_children` here: https://github.com/rapidsai/cudf/blob/5cf7106c1a86d49ae345eff31868343a894c864a/cpp/include/cudf/structs/structs_column_view.hpp#L44-L50

Reply:
The point is that `structs_column_view` should not be an `IS-A` relationship to `column_view`, but a "implemented-in-terms-of" relationship.